pylana/aggregation/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+api for aggregation requests
+"""
+
+from pylana.aggregation.core import AggregationAPI
+from pylana.aggregation.groupings import create_grouping
+from pylana.aggregation.metrics import create_metric
```

Comment:
We don't need to expose these functions to the user any more, so we can delete these two lines here.